buf/registry/module/v1beta1/label.proto

```diff
@@ -71,6 +72,55 @@ message Label {
     (buf.validate.field).required = true,
     (buf.validate.field).string.uuid = true
   ];
+  // Whether or not policy checks are enabled for this Label.
+  //
+  // Policy checks are an enterprise-only feature - contact us to learn more!
+  bool policy_checks_enabled = 10;
+  // The PolicyChecksState for the Commit the Label points to.
+  //
+  // This field will be set if either:
+  //   - Policy checks were enabled on this Label when this Commit was pushed.
+  //   - This Commit transitively depends on a Commit that has a PolicyChecksState.
+  PolicyChecksState policy_checks_state = 11;
+}
+
+// The state of a Commit's policy checks for a particular Label.
+//
+// Policy checks are an enterprise-only feature - contact us to learn more!
+message PolicyChecksState {
```

Comment:
I *really dislike* the plural naming of this - even just `PolicyCheckState` would be better, but I don't know if that fits into your model.

Reply:
I don't feel strongly about this and an open to naming this `PolicyCheckState`.

`PolicyCheckState` implies to me a single overall policy check based on potentially multiple other checks (currently just breaking change detection, but in the future, there could be more checks that roll up into this overall state).

`PolicyChecksState` implies that we may be running multiple independent policy checks (currently only breaking).

Both names work for me so I am fine just renaming this.

Reply:
I'm fine w/ `PolicyCheckState` as long as we are fine with rolling future checks into this (e.g. linting); so basically +1 to Nick's opinion

Reply:
Renamed in f42ba0c440a700b0730f091137aa372e210b60e0